src/core/csv/toa.cc

```diff
@@ -0,0 +1,104 @@
+//------------------------------------------------------------------------------
+// Copyright 2018-2021 H2O.ai
```

Comment:
If this is a new file, shouldn't it be just "Copyright 2021 H2O.ai"?

Reply:
it's new, but it contains code that was previously in `toa.h`, and that was marked as 2018

Reply:
Yeah, I saw it, thought we reflect the date the file was created/updated, not the code.